qa/run_sc_tests.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+set -e -o pipefail
+
+#Log stdout to file, use '| tee /dev/fd/3' to also log to console
+#Must match logfile location referenced in sc_test_framework
+exec 3>&1 1>>"${BASH_SOURCE%/*}/sc_test.log" 2>&1
+
+#Check Environment Variables have been set
+if [[ -z "$BITCOINCLI" ]]; then
+    echo "Environment Variable: BITCOINCLI not set" | tee /dev/fd/3
+    exit 1
+fi
+if [[ -z "$BITCOIND" ]]; then
+    echo "Environment Variable: BITCOIND not set" | tee /dev/fd/3
+    exit 1
+fi
+if [[ -z "$SIDECHAIN_SDK" ]]; then
+    echo "Environment Variable: SIDECHAIN_SDK not set" | tee /dev/fd/3
+    exit 1
+fi
+
+# parse args
+for i in "$@"; do
+  case "${i}" in
+    -extended)
+      EXTENDED="true"
+      shift
+      ;;
+    -exclude=*)
+      EXCLUDE="${i#*=}"
+      shift
+      ;;
+    -split=*)
+      SPLIT="${i#*=}"
+      shift
+      ;;
+    *)
+      # unknown option/passOn
+      passOn+="${i} "
+      ;;
+  esac
+done
+
+#Run the tests
+testScripts=(
+    'websocket_server_fee_payments.py'
```

Comment:
Another point, seems `websocket_server_fee_payments.py` is broken, the same for `websocket_server.py`

Reply:
These tests appear to be broken prior to this change, I have only changed Print() to logging.info() in both these files.  I thought some tests may be failing and that was part of the reason why we wanted all the tests to continue even if one failed

Reply:
seems they were broken for a while. I've tested with the current dev branch.
Will fix them as a separate task.